cnf-certification-test/common/constant.go

```diff
@@ -22,6 +22,7 @@ const (
 	AccessControlTestKey      = "access-control"
 	LifecycleTestKey          = "lifecycle"
 	ChaosTesting              = "chaostesting"
+	Daemonset                 = "daemonset"
```

Comment:
Is this needed?

Reply:
No needed, I was forget to remove it, now I removed.